src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -218,31 +218,75 @@ object AddDatasetMetadataToDeposit {
   }
 
   def createSpatialPoints(dataset: Dataset): Seq[Elem] = {
-    // coordinate order latitude (DCX_SPATIAL_Y), longitude (DCX_SPATIAL_X)
-    dataset.rowsWithValuesForAllOf(composedSpatialPointFields).map(mdKeyValues =>
+    dataset.rowsWithValuesForAllOf(composedSpatialPointFields).map(mdKeyValues => {
+      val srsName = createSrsName(mdKeyValues)
+      // coordinate order x, y = longitude (DCX_SPATIAL_X), latitude (DCX_SPATIAL_Y)
+      val xy = s"${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")}"
+      // coordinate order y, x = latitude (DCX_SPATIAL_Y), longitude (DCX_SPATIAL_X)
+      val yx = s"${mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")} ${mdKeyValues.getOrElse("DCX_SPATIAL_X", "")}"
+
+      val pos = srsName match {
+        case "http://www.opengis.net/def/crs/EPSG/0/28992" => xy
+        case "http://www.opengis.net/def/crs/EPSG/0/4326" => yx
+        case _ => yx
+      }
+
       // @formatter:off
-      <dcx-gml:spatial srsName={createSrsName(mdKeyValues)}>
+      <dcx-gml:spatial srsName={srsName}>
         <Point xmlns="http://www.opengis.net/gml">
-          <pos>{mdKeyValues.getOrElse("DCX_SPATIAL_Y", "")} {mdKeyValues.getOrElse("DCX_SPATIAL_X", "")}</pos>
+          <pos>{pos}</pos>
         </Point>
       </dcx-gml:spatial>
       // @formatter:on
-    )
+    })
   }
 
+  /*
+   Note that X is along North - South and Y is along East - West
+   The lower corner is with the minimal coordinate values and upper corner with the maximal coordinate values
```

Comment:
In this example schematic box it is indeed lower left and upper right, but GML uses the label 'lowerCorner' and 'upperCorner' because you can not be more specific without fixing the direction, which depends on the coordinate system.